libmamba/include/mamba/specs/archive.hpp

```diff
@@ -17,7 +17,8 @@ namespace mamba::specs
 {
     inline static constexpr auto ARCHIVE_EXTENSIONS = std::array{ std::string_view(".tar.bz2"),
                                                                   std::string_view(".conda"),
-                                                                  std::string_view(".whl") };
+                                                                  std::string_view(".whl"),
+                                                                  std::string_view(".tar.gz") };
```

Comment:
there are a number of other potential formats: https://docs.python.org/3.10/distutils/sourcedist.html

Reply:
Cool! Thanks for the pointer :)